shell.nix

```diff
@@ -52,7 +52,7 @@ pkgs.mkShell rec {
     fi
     cmake $unikernel -DARCH=x86_64 -DINCLUDEOS_PACKAGE=${includeos} -DCMAKE_MODULE_PATH=${includeos}/cmake \
                      -DFOR_PRODUCTION=OFF
-    make -j12
+    make -j16
```

Comment:
Perhaps you could use `$NIX_BUILD_CORES` here, see e.g. https://github.com/includeos/IncludeOS/blob/v0.16.0-release/deps/botan/default.nix#L22.

Reply:
Yep, much better - updated.